model-optimizer/mo/front/tf/loader.py

```diff
@@ -261,6 +261,17 @@ def protobuf_attrs(pb:tf_v1.NodeDef):
 
 def protobuf2nx(graph, pb: tf_v1.GraphDef):
     fill_graph_with_nodes(graph, pb.node, get_id=lambda pb: pb.name, get_attrs=protobuf_attrs)
+
+    # Create a library with auxiliary functions used in TensorFlow 2 operations
```

Comment:
nit: "operations" -> "models"?

Reply:
I meant operations such as StalessWhile, StatelessIf, and StatelessSwitch